explainer.md

```diff
@@ -0,0 +1,369 @@
+- [Other Considerations](#other-considerations)
+  - [Initial Experimentation with Stripe](#initial-experimentation-with-stripe)
+  - [Are SPC credentials identical to WebAuthn credentials?](#are-spc-credentials-identical-to-webauthn-credentials)
+  - [Why use the PaymentRequest API](#why-use-the-paymentrequest-api)
+- [Alternatives Considered](#alternatives-considered)
+  - [Traditional WebAuthn](#traditional-webauthn)
+  - [Delegated Authentication](#delegated-authentication)
+- [Security Considerations](#security-considerations)
+  - [Enrollment in cross-origin iframes](#enrollment-in-cross-origin-iframes)
+  - [Cross-origin authentication ceremony](#cross-origin-authentication-ceremony)
+  - [Merchant-supplied data](#merchant-supplied-data)
+- [Privacy Considerations](#privacy-considerations)
+  - [Probing](#probing)
+  - [Credential-sharing](#credential-sharing)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Motivation
+
+Online payments are usually a 3 party interaction:
+
+* The Merchant (e.g. an online web store)
+* The Customer (e.g. the user buying from an online web store)
+* The ??? (e.g. the bank that issued the payment instrument being used)
+
+> **NOTE**: It is not uncommon for there to be a fourth party: the Payment
+> Service Provider, whom the Merchant may delegate the task of accepting the
+> payment method to. This is usually done by the Merchant either redirecting the
+> User to the Payment Service Provider's website, or by embedding the Payment
+> Service Provider in an iframe. Any following mention of the term Merchant
+> should be taken to mean either a Merchant or a Payment Service Provider;
+> whichever is handling payment.
+
+Traditionally, the Customer only interacts with the Merchant, and the Merchant
+communicates with the ??? via a back-channel protocol. However, ???s are
+increasingly looking to authenticate Customers during online payments. This is
+done for both regulatory reasons (e.g. [SCA] in the European Union) or for
+fraud prevention reasons - either fradulent Customers or fradulent Merchants.
+
+Existing methods of authenticating a Customer during an online payment are
+either high friction (e.g. embedding challenge iframes from the ???) or have
+poor user privacy (e.g. fingerprinting or tracking the user for risk analysis,
+to provide a 'frictionless flow'). The payments industry needs a consistent,
+low friction, strong authentication flow for online payments.
+
+The [Web Authentication (WebAuthn) API][webauthn] makes FIDO-based
+authentication available on the web, which provides a strong, low-friction
+method for a user to prove a pre-established identity with a given site.
+However, WebAuthn is not immediately suitable to solve the payments
+authentication problem, as:
+
+1. It requires the ??? to be present in the transaction (e.g. via an embedded
+   iframe), which increases user friction and lowers conversion rates.
```

Comment:
Why does having the bank in an iframe increase user friction? I think that redirecting the user away from the merchant site to the bank site does, but it's not clear to me why the iframe approach does. It might raise security issues (cf that and other rationale from Stripe for moving away from iframes for bank code).

Reply:
I think this is somewhere I would like to hear from @jcemer-stripe on. I find it easy to articulate why FIDO authentication is a win for payments (so much easier than OTP), and why payment info in the cryptogram is a win (help out dynamic linking), but when trying to justify the 3p authentication ceremony abilities I come up somewhat short in justifications.

If @jcemer-stripe can provide better justification, with super bonus points for publicly available data, that would be very useful both for this explainer, SPC in general, and our upcoming chat with WebAuthn I think.

Reply:
@stephenmcgruer, see some information provided during task force discussions by Benjamin:
https://lists.w3.org/Archives/Public/public-webauthn-pay/2020Jan/0002.html

Reply:
I'd added a TODO comment to incorporate this information, though its still all opinion that I can tell (i.e. no data). Which is still valuable, of course :)